Marlin/src/lcd/menu/menu_bed_corners.cpp

```diff
@@ -346,20 +368,44 @@ static void _lcd_level_bed_corners_homing() {
   #endif
 }
 
-void _lcd_level_bed_corners() {
-  ui.defer_status_screen();
-  if (!all_axes_trusted()) {
-    set_all_unhomed();
-    queue.inject_P(G28_STR);
+#if HAS_STOWABLE_PROBE && DISABLED(BLTOUCH)
+void _deploy_probe() {
+  if (ui.should_draw()) {
+    constexpr uint8_t line = (LCD_HEIGHT - 1) / 2;
+    MenuItem_static::draw(line, GET_TEXT_F(MSG_MANUAL_DEPLOY));
   }
+}
 
-  // Disable leveling so the planner won't mess with us
-  #if HAS_LEVELING
-    leveling_was_active = planner.leveling_active;
-    set_bed_leveling_enabled(false);
+void deploy_probe() {
+  if (!corner_probing_done) probe.deploy(true);
+  ui.goto_screen([]{
+    _deploy_probe();
```

Comment:
Since this is within a screen handler, `_deploy_probe` will be called repeatedly. Then there is nothing within this screen handler that exits the screen handler, so it will keep doing this until the code below invokes `_lcd_level_bed_corners_homing` to potentially set some new screen handler.